src/test/java/com/amihaiemil/docker/RtContainerITCase.java

```diff
@@ -58,4 +62,80 @@ public void renamesContainer() throws Exception {
         );
         container.remove();
     }
+
+    /**
+     * {@link RtContainer} can start/stop Docker container it represents.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public void startStopContainer() throws Exception {
+        final Container container = new LocalDocker(
+            new File("/var/run/docker.sock")
```

Comment:
@bkuzmic These path does not works on windows machines.

Reply:
@paulodamaso no need for puzzles, I think. I would say that we assume by default that ``LocalDocker`` will only work on linux -- it uses ``Unix Sockets`` behind, which I don't think are available on Windows :)

Reply:
@amihaiemil Okay then, that's good for me